src/exercises/fanout/consumer.ts

```diff
@@ -0,0 +1,38 @@
+import { connect } from "amqplib";
+import { random } from "lodash";
+
+random();
+
+export function wait(timeout: number) {
+  return new Promise((res, _) => {
+    return setTimeout(res, timeout);
+  });
+}
+async function main() {
+  const connection = await connect("amqp://localhost");
+  const channel = await connection.createChannel();
+  const exchangeName = "fanoutExchange";
+  await channel.assertExchange(exchangeName, "fanout");
+  //in case multiple consumer instances are created,
+  //each instance will have a temporary queue from which it will receive all messages sent by the producer
+  const { queue } = await channel.assertQueue("", {
+    autoDelete: true,
+    durable: false,
+  });
```

Comment:
```suggestion
  const { queue } = await channel.assertQueue("", { autoDelete: true, durable: false});
```

In general try to write on single line